.github/workflows/release.yaml

```diff
@@ -0,0 +1,21 @@
+name: Latest Release
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+    test-job:
+      name: "Tag / Build / Push Container"
+      runs-on: ubuntu-latest
+      steps:
+      - name: Trigger Buildkite Pipeline
+        uses: buildkite/trigger-pipeline-action@v1.2.0
+        env:
+          BUILDKITE_API_ACCESS_TOKEN: ${{ secrets.Buildkite_Token }}
+          PIPELINE: ${{ github.repository }}
+          COMMIT: "HEAD"
+          BRANCH: "master"
+          MESSAGE: ":github: Triggered from a GitHub Action"
+          BUILD_ENV_VARS: "{\"BUILDKITE_TAG\": \"latest\"}"
```

Comment:
This should be the Git tag, not `latest`

Reply:
Also, there should be a specific env for the `RELEASE_VERSION`, and subsequent changes to the build script, to ensure a random tag build doesn't get the `latest` Docker tag (for example). It's still possible to manually create a tag build, or something as simple as the Buildkite filter rules changing would break this. All releases have tags, but not all tags are releases.

Reply:
Fixed this as discussed, using GITHUB_EVENT_NAME and GITHUB_REF.